contracts/HybridCustody.cdc

```diff
@@ -716,6 +718,18 @@ pub contract HybridCustody {
             self.resources <- {}
         }
 
+        /// Returns a capability to this child account's CapabilityFilter
+        ///
+        pub fun getCapabilityFilter(): Capability<&{CapabilityFilter.Filter}> {
+            return self.filter
+        }
+
+        /// Returns a capability to this child account's CapabilityFactory
+        ///
+        pub fun getCapabilityFactory(): Capability<&CapabilityFactory.Manager{CapabilityFactory.Getter}> {
```

Comment:
Naming here might lead someone to believe they can retrieve a specific `Factory` implementation.
Thoughts about `getCapabilityFactoryManager()` or `getFactoryManager()`?

Reply:
No strong opinions here, i'll adjust this tomorrow